cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,58 @@
-/// <reference types='cypress' />
-/// <reference types='../support' />
+/// <reference types="cypress" />
+/// <reference types="../support" />
+
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject';
+
+const settingsPage = new SettingsPageObject();
+const signInPage = new SignInPageObject();
 
 describe('Settings page', () => {
-  before(() => {
+  let user;
 
+  before(() => {
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
   });
 
   beforeEach(() => {
-
+    cy.task('db:clear');
+    cy.visit('/');
+    cy.register();
+    signInPage.visit();
+    signInPage.typeEmail('riot@qa.team');
+    signInPage.typePassword('12345Qwert!');
+    signInPage.clickSignInBtn();
+    settingsPage.clickProfileLink();
   });
 
   it('should provide an ability to update username', () => {
-
+    settingsPage.typeUsername(user.username);
+    settingsPage.submitForm();
+    settingsPage.assertProfilePage(user.username);
   });
 
   it('should provide an ability to update bio', () => {
-
+    settingsPage.typeBio(user.bio);
+    settingsPage.submitForm();
+    settingsPage.assertProfilePage(user.bio);
```

Comment:
Ensure that the `assertProfilePage` method in `SettingsPageObject` correctly checks for the updated bio. It should confirm that the profile page displays the new bio.